lib/pg_eventstore/client.rb

```diff
@@ -109,6 +109,27 @@ def read(stream, options: {}, middlewares: nil)
         call(stream, options: { max_count: config.max_count }.merge(options))
     end
 
+    # Links event from one stream into another stream. You can later access it by providing :resolve_link_tos option
+    # when reading from a stream. Only existing events can be linked.
+    # @param stream [PgEventstore::Stream]
+    # @param events_or_event [PgEventstore::Event, Array<PgEventstore::Event>]
+    # @param options [Hash]
+    # @option options [Integer] :expected_revision provide your own revision number
+    # @option options [Symbol] :expected_revision provide one of next values: :any, :no_stream or :stream_exists
```

Comment:
Don't you want to squeeze it into one option like `option options [Integer, Symbol] :expected_revision provide your own revision integer number or one of next values: :any, :no_stream or :stream_exists` ?

Reply:
Although I saw you split them everywhere so lets not change it.

Reply:
Having it separate is more readable